### PR TITLE
Add `bell-dot` icon and refine `bell-*` icons

### PR DESCRIPTION
**Pull request by danielbayley**
Alternatives explored:

image

### CODE REVIEW

**Comment by danielbayley** [comment]
> Hey @danielbayley, Can you provide these alternative at a 100% view?

Sure:

Untitled

**Comment by karsa-mistmere** [comment]
Let's first wait for @ericfennis' two cents on this, to me the 3rd row looks the most balanced.

File: icons/bell-dot.svg [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M18.6 13c.9 3 2.4 4 2.4 4H3s3-2 3-9c0-3.3 2.7-6 6-6h.3" />
+  <path d="M10.3 21c.3.6 1 1 1.7 1s1.4-.4 1.7-1" />
+  <circle cx="18" cy="6" r="3" />
```
Comment (jguddas) [inline icons/bell-dot.svg:14]:
```suggestion
  <circle cx="18" cy="6" r="3"/>
  <path d="M18.6 13a7.4 7.4 0 0 0 2.4 4H3s3-2 3-9a6.1 6.1 0 0 1 6.3-6"/>
  <path d="M10.3 21a1.94 1.94 0 0 0 3.4 0"/>
```

File: icons/bell-minus.svg [hunk 2]
```diff
@@ -9,7 +9,7 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M13.73 21a2 2 0 0 1-3.46 0" />
-  <path d="M21 5h-6" />
-  <path d="M18.021 9C18.29 15.193 21 17 21 17H3s3-2 3-9a6 6 0 0 1 7-5.916" />
+  <path d="M18.1 10c.5 5.4 2.9 7 2.9 7H3s3-2 3-9c0-3.3 2.7-6 6-6 .6 0 1.1.1 1.7.2" />
+  <path d="M10.3 21c.3.6 1 1 1.7 1s1.4-.4 1.7-1" />
+  <path d="M21 6h-6" />
```
Comment (jguddas) [inline icons/bell-minus.svg:14]:
```suggestion
  <path d="M18.1 10a9.3 9.3 0 0 0 2.9 7H3s3-2 3-9a6 6 0 0 1 7.7-5.8"/>
  <path d="M10.3 21a1.94 1.94 0 0 0 3.4 0"/>
  <path d="M21 6h-6"/>
```

File: icons/bell-off.svg [hunk 3]
```diff
@@ -9,9 +9,8 @@
...
   stroke-linejoin="round"
 >
-  <path d="M13.73 21a2 2 0 0 1-3.46 0" />
-  <path d="M18.63 13A17.888 17.888 0 0 1 18 8" />
-  <path d="M6.26 6.26A5.86 5.86 0 0 0 6 8c0 7-3 9-3 9h14" />
-  <path d="M18 8a6 6 0 0 0-9.33-5" />
+  <path d="M8.7 3c1-.6 2.1-1 3.3-1 3.3 0 6 2.7 6 6 0 2.1.3 3.7.6 5" />
+  <path d="M17 17H3s3-2 3-9c0-.6.1-1.2.3-1.7" />
   <path d="m2 2 20 20" />
+  <path d="M10.3 21c.3.6 1 1 1.7 1s1.4-.4 1.7-1" />
```
Comment (jguddas) [inline icons/bell-off.svg:15]:
```suggestion
  <path d="M8.7 3A6.02 6.02 0 0 1 18 8a21.3 21.3 0 0 0 .6 5"/>
  <path d="M17 17H3s3-2 3-9a4.67 4.67 0 0 1 .3-1.7"/>
  <path d="m2 2 20 20"/>
  <path d="M10.3 21a1.94 1.94 0 0 0 3.4 0"/>
```

**Review by ericfennis — APPROVED** [review]
I agree with @karsa-mistmere, LGTM!

File: icons/bell.svg [hunk 4]
```diff
@@ -9,6 +9,6 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M18 8A6 6 0 0 0 6 8c0 7-3 9-3 9h18s-3-2-3-9" />
-  <path d="M13.73 21a2 2 0 0 1-3.46 0" />
+  <path d="M6 8c0-3.3 2.7-6 6-6s6 2.7 6 6c0 7 3 9 3 9H3s3-2 3-9z" />
+  <path d="M10.3 21c.3.6 1 1 1.7 1s1.4-.4 1.7-1" />
```
Comment (karsa-mistmere) [inline icons/bell.svg:13]:
```suggestion
  <path d="M6 8a6 6 0 0 1 12 0c0 7 3 9 3 9H3s3-2 3-9"/>
  <path d="M10.3 21a1.94 1.94 0 0 0 3.4 0"/>
```

File: icons/bell-ring.svg [hunk 5]
```diff
@@ -9,8 +9,8 @@
...
   stroke-linejoin="round"
 >
-  <path d="M18 8A6 6 0 0 0 6 8c0 7-3 9-3 9h18s-3-2-3-9" />
-  <path d="M13.73 21a2 2 0 0 1-3.46 0" />
-  <path d="M2 8c0-2.2.7-4.3 2-6" />
-  <path d="M22 8a10 10 0 0 0-2-6" />
+  <path d="M6 8c0-3.3 2.7-6 6-6s6 2.7 6 6c0 7 3 9 3 9H3s3-2 3-9z" />
+  <path d="M10.3 21c.3.6 1 1 1.7 1s1.4-.4 1.7-1" />
+  <path d="M4 2C2.8 3.7 2 5.7 2 8" />
+  <path d="M22 8c0-2.3-.8-4.3-2-6" />
```
Comment (karsa-mistmere) [inline icons/bell-ring.svg:15]:
```suggestion
  <path d="M6 8a6 6 0 0 1 12 0c0 7 3 9 3 9H3s3-2 3-9"/>
  <path d="M10.3 21a1.94 1.94 0 0 0 3.4 0"/>
  <path d="M4 2C2.8 3.7 2 5.7 2 8"/>
  <path d="M22 8c0-2.3-.8-4.3-2-6"/>
```

File: icons/bell-plus.svg [hunk 6]
```diff
@@ -9,8 +9,8 @@
...
   stroke-linejoin="round"
 >
-  <path d="M18.387 12C19.198 15.799 21 17 21 17H3s3-2 3-9a6 6 0 0 1 7-5.916" />
-  <path d="M13.73 21a2 2 0 0 1-3.46 0" />
-  <path d="M18 2v6" />
-  <path d="M21 5h-6" />
+  <path d="M19.3 14.8C20.1 16.4 21 17 21 17H3s3-2 3-9c0-3.3 2.7-6 6-6 1 0 1.9.2 2.8.7" />
+  <path d="M10.3 21c.3.6 1 1 1.7 1s1.4-.4 1.7-1" />
+  <line x1="18" x2="18" y1="5" y2="11" />
+  <line x1="15" x2="21" y1="8" y2="8" />
```
Comment (karsa-mistmere) [inline icons/bell-plus.svg:15]:
```suggestion
  <path d="M10.3 21a1.94 1.94 0 0 0 3.4 0"/>
  <path d="M15 8h6"/>
  <path d="M18 5v6"/>
```

File: icons/bell-off.svg [hunk 7]
```diff
@@ -9,9 +9,8 @@
...
   stroke-linejoin="round"
 >
-  <path d="M13.73 21a2 2 0 0 1-3.46 0" />
-  <path d="M18.63 13A17.888 17.888 0 0 1 18 8" />
-  <path d="M6.26 6.26A5.86 5.86 0 0 0 6 8c0 7-3 9-3 9h14" />
-  <path d="M18 8a6 6 0 0 0-9.33-5" />
+  <path d="M8.7 3A6.02 6.02 0 0 1 18 8a21.3 21.3 0 0 0 .6 5" />
+  <path d="M17 17H3s3-2 3-9a4.67 4.67 0 0 1 .3-1.7" />
   <path d="m2 2 20 20" />
+  <path d="M10.3 21a1.94 1.94 0 0 0 3.4 0" />
```
Comment (karsa-mistmere) [inline icons/bell-off.svg:15]:
```suggestion
  <path d="M8.7 3A6 6 0 0 1 18 8a21.3 21.3 0 0 0 .6 5"/>
  <path d="M17 17H3s3-2 3-9a4.67 4.67 0 0 1 .3-1.7"/>
  <path d="M10.3 21a1.94 1.94 0 0 0 3.4 0"/>
  <path d="m2 2 20 20"/>
```

File: icons/bell-minus.svg [hunk 8]
```diff
@@ -9,7 +9,7 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M13.73 21a2 2 0 0 1-3.46 0" />
-  <path d="M21 5h-6" />
-  <path d="M18.021 9C18.29 15.193 21 17 21 17H3s3-2 3-9a6 6 0 0 1 7-5.916" />
+  <path d="M18.4 12c.8 3.8 2.6 5 2.6 5H3s3-2 3-9c0-3.3 2.7-6 6-6 1.8 0 3.4.8 4.5 2" />
+  <path d="M10.3 21c.3.6 1 1 1.7 1s1.4-.4 1.7-1" />
+  <line x1="15" x2="21" y1="8" y2="8" />
```
Comment (karsa-mistmere) [inline icons/bell-minus.svg:14]:
```suggestion
  <path d="M18.4 12c.8 3.8 2.6 5 2.6 5H3s3-2 3-9c0-3.3 2.7-6 6-6 1.8 0 3.4.8 4.5 2" />
  <path d="M10.3 21a1.94 1.94 0 0 0 3.4 0"/>
  <path d="M15 8h6"/>
```

File: icons/bell-dot.svg [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M19.4 14.9C20.2 16.4 21 17 21 17H3s3-2 3-9c0-3.3 2.7-6 6-6 .7 0 1.3.1 1.9.3" />
+  <path d="M10.3 21c.3.6 1 1 1.7 1s1.4-.4 1.7-1" />
+  <circle cx="18" cy="8" r="3" />
```
Comment (karsa-mistmere) [inline icons/bell-dot.svg:14]:
```suggestion
  <path d="M19.4 14.9C20.2 16.4 21 17 21 17H3s3-2 3-9c0-3.3 2.7-6 6-6 .7 0 1.3.1 1.9.3" />
  <path d="M10.3 21a1.94 1.94 0 0 0 3.4 0"/>
  <circle cx="18" cy="8" r="3" />
```

**Review by karsa-mistmere — CHANGES_REQUESTED** [review]
Some optimizations could still be done.

**Review by karsa-mistmere — APPROVED** [review]

**Review by ericfennis — APPROVED** [review]

**Pull request merged**